Flutter/calendar/builders.md

```diff
@@ -328,7 +328,7 @@ The [ResourceViewHeaderBuilder](https://pub.dev/documentation/syncfusion_flutter
 
 
 {% tabs %}
-{% highlight Dart %}
+{% highlight dart hl_lines="7 8" %}
```

Comment:
Not Highlighted

https://help.syncfusion.com/flutter/calendar/right-to-left#changing-the-locale-to-rtl-languages

Reply:
Highlighted the mentioned topics.